### ISSUE TITLE
How did you fix your posterior chain?

### ISSUE THREAD

**Issue by abtreece** [jasoncwarner/ama#35]
In [The Changelog - Episode 395](https://changelog.com/podcast/395), which is easily one of the best podcast episodes I've heard in a while, you mentioned a half finished blog post that would probably never be published about how you wrecked and fixed your back. I am also in my 40s and the ebb and flow of my career has frequently interrupted my focus on personal health. I have back, hip, and hamstring issues and feel as if I am reaching a point of no return. I would really like to read that blog post, but in lieu of it being published, would you provide details on that process here?

Linking to Issue #9 because of the relevance of the home gym discussion.

**Comment by jasoncwarner** [comment]
Oh, this topic. It's super involved and multi-faceted, and unfortunately there is probably no way to know *exactly* what is happening with you so my advise will be of the general variety. 

And, of course, the caveat. I'm not a doctor. Please talk to a doctor or someone with more knowledge than me. What I'm going to give is information that worked for me, and even that information is stuff that I'm adapting as I go to make more changes and fixes. 

Few notes specific to me that also might apply to you. 

1. Core strength, stamina and stability is a major factor in back pain. 

2. Tightness *can* be, though in my case it didn't seem to be. 

3. Core strength and overall strength are probably related but not as much as you think. 

4. Muscle stamina, particularly the QL, was probably the largest culprit for me to figure out. 

What I found through experimentation is that when I'm in pain, my [QL](https://en.wikipedia.org/wiki/Quadratus_lumborum_muscle) would essentially be "tired" and "not firing" correctly. If the body is a closed but very complex system and the muscles all around the body need to operate efficiently and in sync, my QL were not cooperating. What was/is happening is that I would do whatever and eventually find that my QL had effectively gotten worn out from the day or exercise or whatever and "stopped firing" which meant other muscles would start taking over their job function....

This would lead to my glutes, obliques, and .... lower back doing more work than they should. And in my case, the low back would start to shut down next as *they* got worn out. 

What to do about it in my case? 

Work on overall strength *and* stamina for the various muscles. *AND* find ways to "fire up" the QL once they had shut down.

**Comment by jasoncwarner** [comment]
First, I love the [12 minute foundation workout](https://www.youtube.com/watch?v=4BOTvaRaDjI) for yoga-like stretching and strengthening.

**Comment by jasoncwarner** [comment]
The [McGill Big 3](https://squatuniversity.com/2018/06/21/the-mcgill-big-3-for-core-stability/) is where most people in the fitness community recommend you turn when starting out.

**Comment by jasoncwarner** [comment]
Next, I started to work on overall core strength, stamina and specifically the QL. 

For the QL warmup and strengthening, I really really really like [the butt walk](https://www.youtube.com/watch?v=CyVXiCQ6V2I&t=1s). 

These are from [Donnie Thompson](https://www.instagram.com/thompsonbowtie/) though I can't find the original video for them. Donnie is an old school powerlifter and has some of the best for free advise out there. I highly recommend following him. 

[This is a great overall QL specific video](http://chadwaterbury.com/weakness-in-this-muscle-will-rob-your-power/) and worth looking at if this is true of you.

**Comment by jasoncwarner** [comment]
For core strength and stamina (including back and glutes) I found that the best thing for me was natural movement patterns. 

I was fortunate enough to have worked with [Darryl Edwards](https://www.amazon.com/Darryl-Edwards/e/B00CA15V10/ref=dp_byline_cont_book_1) on a [Paleo Fitness](https://www.amazon.com/Paleo-Fitness-Training-Nutrition-Program/dp/1612431658/). Darryl is also a [huge advocate for both natural and animal moves](https://www.amazon.com/gp/product/B07BLS29DH/ref=dbs_a_def_rwt_hsch_vapi_tkin_p1_i0). I ended up adopting lots of these into my daily practice. 

The few that worked the best for me. 

1. bear crawl
2. crab walk

[Here is a good walk through on a bunch of animal patterns/walks/crawls](https://www.wellandgood.com/animal-walk-exercises/). You can find lots of information out there on these.

**Comment by jasoncwarner** [comment]
And I can't end without talking about planks. 

The side plank is my absolute go to. If ever in pain, I try to do some butt walks, followed by a bear crawl, crab walk and then a few minutes of side planks. I want to get in some plank variant every single day too. 

And I absolutely love [this 8 minute plank brutal workout for overall strength](https://www.youtube.com/watch?v=_lfR4sl0ZCE).

**Comment by jasoncwarner** [comment]
Last comment. I also found I had some hip mobility issues and there is literally only one thing that really helped....

the 90 90 hip stretch. I like to do this with static 2 minute holds....literally a ton of information on this one but [this is a very very good video on it](https://www.youtube.com/watch?v=uxh_b87er9s). 

I learned about this one from [John Quint](https://www.instagram.com/jquintnmt/?hl=en) who I saw for some physical therapy. He's based out of Columbus Ohio but [is available if you want to talk to him or get sessions from him](http://johnquintnmt.com/about-me/). I feel lucky to have found him too as he does group sessions for folks like me and personal session for professional athletes.
